app/src/main/res/values/strings.xml

```diff
@@ -1820,8 +1820,10 @@
     <!-- Title of an app feature. Smaller than a heading.-->
     <string name="pocket_stories_feature_title">Powered by Pocket.</string>
     <!-- Caption for describing a certain feature. The placeholder is for a clickable text (eg: Learn more) which will load an url in a new tab when clicked.  -->
-    <string name="pocket_stories_feature_caption">Part of the Firefox family. %s</string>
-    <!-- Clickable text for opening an external link for more information about Pocket. -->
+    <string name="pocket_stories_feature_caption" moz:RemovedIn="106" tools:ignore="UnusedResources">Part of the Firefox family. %s</string>
+    <!-- Caption for describing the Pocket stories feature. This is the first part of the string with "pocket_stories_feature_learn_more" following on the same line. -->
+    <string name="pocket_stories_feature_caption_2">Part of the Firefox family.</string>
+    <!-- Clickable text for opening an external link for more information about Pocket. Follows "pocket_stories_feature_caption_2" on the same line. -->
```

Comment:
Can you also move this string outside of the PR which has been focusing on adding test tags? We should try to avoid hidden changes like this from being grouped with other intended changes.

Reply:
Confirming that it's better to leave the placement of "Learn more" up to localizers, since this may vary depending on the language. So better to keep this as a single string

Reply:
The strings not being spitted started the discussion about `testTag`s.
We need two separate strings so that UI tests can act on each separately and specifically allow UI tests to click on the "Learn more" link.
If the "Learn more" link is a separate composable it would be easy to interact with it.
Otherwise after many tries is was deemed impossible.

Reply:
Then I would simply say we can't test the "Learn more" link. It's simply incorrect to assume we can append 2 different strings incorrectly from a localization point of view. The "%s" will typically appear at the beginning for RTL languages, and there is also the possibility that it could appear in the middle for some other languages depending on how it is localized. The way it is hardcoded to append at the end is only correct for some language, but not the rest.

Reply:
Dropped splitting the string.